backend/open_webui/models/message_metrics.py

```
@@ -0,0 +1,658 @@
+                if domain:
+                    query = query.filter(MessageMetric.user_domain == domain)
+                result = query.with_entities(
+                    func.sum(MessageMetric.total_tokens),
+                ).first()
+                return round(result[0], 2) if result and result[0] else 0
+        except Exception as e:
+            logger.error(f"Failed to get message tokens number: {e}")
+            return 0  # Return 0 instead of None
+
+    def get_daily_message_tokens_sum(
+        self, days: int = 1, domain: Optional[str] = None
```

Ruff (0.12.2)

Unused method argument: `days`

(ARG002)